### PR TITLE
Fix: Mistypography of UUD 1945

### PR DESCRIPTION
**Pull request by muhsatrio**
Change `UUD 1945` to `Pembukaan UUD 1945` in `daftar isi`

@mazipan

### CODE REVIEW

File: data/daftar-isi.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default [
         api: '/api/pancasila',
       },
       {
-        text: 'UUD 1945',
+        text: 'Pembukaan UUD 1945',
         href: '/uud1945/',
```
Comment (mazipan) [inline data/daftar-isi.js:12]: Do we also need to change the slug?

Comment (muhsatrio) [reply data/daftar-isi.js:12]: aaa right! wait a minute, I'll fix it too

File: data/daftar-isi.js [hunk 2]
```diff
@@ -8,8 +8,8 @@ export default [
         api: '/api/pancasila',
       },
       {
-        text: 'UUD 1945',
-        href: '/uud1945/',
+        text: 'Pembukaan UUD 1945',
+        href: '/pembukaan-uud-1945/',
```
Comment (mazipan) [inline data/daftar-isi.js:12]: Can you also help to find in all directory for word `/uud1945`, I guess there are other places that using this slug.

Comment (muhsatrio) [reply data/daftar-isi.js:12]: pushed, kindly check again

**Review by mazipan — APPROVED** [review]
Looks good 🐐

**Pull request merged**